CMakeLists.txt

```diff
@@ -53,4 +61,6 @@ message(
     "-- Setting: DOCKER_PREFIX=${DOCKER_PREFIX}\n"
     "-- Setting: UPDATE_DOCKERFILES=${UPDATE_DOCKERFILES}\n"
     "-- Setting: UPDATE_IMAGELIST=${UPDATE_IMAGELIST}\n"
+    "-- Setting: UPDATE_IMAGE_README=${UPDATE_IMAGE_README}\n")
```

Comment:
** Pls. remove Right bracket in this line. Error would be found when running cmake**
CMake Error: Error in cmake code at
/home/vcse/ovc/test_pr/Dockerfiles/CMakeLists.txt:65:
Parse error.  Expected a command name, got quoted argument with text "-- Setting: UPDATE_DOCKERHUB_README=${UPDATE_DOCKERHUB_README}\n".
-- Configuring incomplete, errors occurred!